### PR TITLE
fixed reviver implant not working

### PR DESCRIPTION
**Pull request by updooter**
## About The Pull Request
Fixes the reviver implant not working, which should check if the user is in crit

## Why It's Good For The Game

Its a fix

## Testing Photographs and Procedure

</details>

## Changelog
:cl:
fix: fixed the reviver implant not working
/:cl:

### CODE REVIEW

File: code/modules/surgery/organs/augments_chest.dm [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
 /obj/item/organ/cyberimp/chest/reviver/on_life()
 	if(reviving)
-		if(owner.stat == UNCONSCIOUS)
+		if(owner.stat == UNCONSCIOUS || owner.stat == HARD_CRIT || owner.stat == SOFT_CRIT)
```
Comment (PowerfulBacon) [inline code/modules/surgery/organs/augments_chest.dm]: Just check for owner.stat
if(owner.stat)

File: code/modules/surgery/organs/augments_chest.dm [hunk 2]
```diff
@@ -66,7 +66,7 @@
 
 	if(cooldown > world.time)
 		return
-	if(owner.stat != UNCONSCIOUS)
+	if(!(owner.stat))
```
Comment (PowerfulBacon) [inline code/modules/surgery/organs/augments_chest.dm]:
```suggestion
	if(!owner.stat)
```
No need for brackets here

**Review by PowerfulBacon — APPROVED** [review]

**Comment by updooter** [comment]
please approve the workflow testing

**Comment by updooter** [comment]
when is it going to be merged?

**Comment by PowerfulBacon** [comment]
We have a merge policy that requires PRs to be open for 24h or have 2 people approve it first.

**Review by itsmeow — APPROVED** [review]

**Pull request merged**